udfs/community/parse_timestamp.sqlx

```diff
@@ -0,0 +1,31 @@
+config { hasOutput: true }
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+-- Safely allows greater than comparison on numbers of any magnitude
+CREATE OR REPLACE FUNCTION ${self()}(f STRING, s STRING)
```

Comment:
Thoughts on incorporating the word "python" into the UDF name so that users know it's a python parsing function?